src/framework/theme/components/autocomplete/autocomplete.component.spec.ts

```diff
@@ -0,0 +1,273 @@
+import { Component, EventEmitter, Output, QueryList, ViewChild, ViewChildren } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { NB_DOCUMENT } from '@nebular/theme/theme.options';
+import { By } from '@angular/platform-browser';
+import {
+  NbOptionComponent,
+  NbAutocompleteModule,
+  NbOptionListModule,
+  NbThemeModule,
+  NbLayoutModule,
+  NbOverlayContainerAdapter,
+} from '@nebular/theme';
+import { NbAutocompleteComponent } from './autocomplete.component';
```

Comment:
This components must be in public api, so why don't import them from `@nebular/theme`. This way we'll ensure components are exported

Reply:
Agree, fixed